### PR TITLE
Feature: Add color picker component for accent color setting

### PR DESCRIPTION
**Pull request by SpecialAro**
#### Pre-flight Checklist

1. Please remember that if you are logging a bug for some service that has _stopped working_ or is _working incorrectly_, please log the bug [here](https://github.com/ferdium/ferdium-recipes/issues)
2. If you are requesting support for a **new service** in Ferdium, please log it [here](https://github.com/ferdium/ferdium-recipes/pulls)
3. Please remember to read the [self-help documentation](https://github.com/ferdium/ferdium-app#troubleshooting-recipes-self-help) - in case it helps you unblock yourself for issues related to older versions of recipes that were installed on your machine. (These will get automatically upgraded when you upgrade to the newer versions of Ferdium, but to get new recipes between Ferdium releases, this documentation is quite useful.)
4. Please ensure you've completed all of the following.

- [x] I have read the [Contributing Guidelines](https://github.com/ferdium/ferdium-app/blob/develop/CONTRIBUTING.md) for this project.
- [x] I agree to follow the [Code of Conduct](https://github.com/ferdium/ferdium-app/blob/develop/CODE_OF_CONDUCT.md) that this project adheres to.
- [x] I have searched the [issue tracker](https://github.com/ferdium/ferdium-app/issues) for a feature request that matches the one I want to file, without success.

#### Description of Change
Feature: Add color picker component for accent color setting

#### Motivation and Context
My OCD lead me to add a color picker that automatically updates the input field of the CSS code so that people can just slide the picker and chose the color, while maintaining the ability of putting a CSS color code. Also, added a "Apply" button for this setting so that the color is updated when clicked.

There is probably some cleaning to do on all the files, but given it is my first time doing something of react specific stuff I'll need some guidance and assistance on that. Please review them and suggest things 🙏 

#### Screenshots
| Before image | After image |
|--- |--- |

#### Checklist

- [x] My pull request is properly named
- [x] The changes respect the code style of the project (`npm run prepare-code`)
- [x] `npm test` passes
- [x] I tested/previewed my changes locally

#### Release Notes
Feature: Add color picker component for accent color setting

### CODE REVIEW

File: src/components/ui/ColorPickerInput.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
+import { Component } from 'react';
```
Comment (vraravam) [inline src/components/ui/ColorPickerInput.js]: Can you please convert this file to typescript?

**Review by vraravam — CHANGES_REQUESTED** [review]

File: src/components/ui/ColorPickerInput.js (hunk 1 above)
Comment (SpecialAro) [reply src/components/ui/ColorPickerInput.js:1]: Yes, thought about it. I'm still learning proper typescript so hopefully I can convert this file successfully. Will try yo do it as soon as I can. 😊

Comment (SpecialAro) [reply src/components/ui/ColorPickerInput.js]: Yes, thought about it. I'm still learning proper typescript so hopefully I can convert this file successfully. Will try to do it as soon as I can. 😊

Comment (vraravam) [reply src/components/ui/ColorPickerInput.js]: I have converted that file into typescript. Please review and run locally. If it's good, then please merge. Thanks!

**Review by vraravam — APPROVED** [review]

**Pull request merged**